conf/repo/logistics.yml

```diff
@@ -145,6 +145,16 @@ stock-logistics-workflow:
   name: Odoo stock, workflow and organization
   psc: logistics-maintainers
   psc_rep: logistics-maintainers-psc-representative
+stock-weighing:
+  branches:
+    - "15.0"
+    - "16.0"
+    - "17.0"
+  category: Stock
+  default_branch: "15.0"
+  name: Shopfloor for weighing operations
+  psc: stock-weighing
```

Comment:
@chienandalu you need to reuse the same code used for the psc role. I fixed it directly on master.